File: frontend/src/types/services.ts
export type ServiceKey = "emby" | "sonarr" | "radarr" | "qbittorrent" | "cross_seed" | "seer"

export interface ServiceStatus {
  service: ServiceKey
  // Nom affiché : « Emby »/« Jellyfin », nom de l'instance Sonarr/Radarr...
  name: string
  ok: boolean
  message: string
}

export interface ServicesStatus {
  checked_at: string
  services: ServiceStatus[]
}

File: backend/app/schemas/services.py
from datetime import datetime

from pydantic import BaseModel


class ServiceStatusRead(BaseModel):
    service: str  # emby | sonarr | radarr | qbittorrent | cross_seed | seer
    # Nom affiché : « Emby »/« Jellyfin », nom de l'instance Sonarr/Radarr...
    name: str
    ok: bool
    message: str


class ServicesStatus(BaseModel):
    checked_at: datetime
    services: list[ServiceStatusRead]
